<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>631</width>
      <height>151</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>메뉴</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>절대 경로</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>71</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Time</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Domain</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DNS_IP</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>631</width>
      <height>521</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>연번</string>
     </property>
     <property name="background">
      <color>
       <red>200</red>
       <green>199</green>
       <blue>199</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>시간</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>대상 도메인</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IP 주소</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DNS 서버 IP</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>690</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0개</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
